plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/flex/AppEngineFlexFacetTest.java

```diff
@@ -0,0 +1,12 @@
+package com.google.cloud.tools.eclipse.appengine.facets.flex;
```

Comment:
Is there a reason for an extra package? I would think one package for the different facets is enough.